packages/datadog-plugin-oracledb/src/index.js

```diff
@@ -14,7 +15,13 @@ class OracledbPlugin extends Plugin {
 
     this.addSub('apm:oracledb:execute:start', ({ query, connAttrs }) => {
       const service = getServiceName(this.tracer, this.config, connAttrs)
-      const connectStringObj = new URL('http://' + connAttrs.connectString)
+      let connectStringObj
+      try {
+        connectStringObj = new URL(`http://${connAttrs.connectString}`)
+      } catch (e) {
+        connectStringObj = new URL(`http://localhost:1521/xepdb1`)
```

Comment:
This is giving incorrect information. No information is better than incorrect information.

Reply:
We could change the tagging logic to exclude the tags we don't have the information for.